test/cppsuite/src/main/test.cpp

```diff
@@ -94,9 +94,8 @@ test::run()
     bool enable_logging, statistics_logging;
     configuration *statistics_config;
     std::string statistics_type;
-    /* Build the database creation config string. */
-    std::string db_create_config = CONNECTION_CREATE;
-
+    /* Build the database creation config string. Allow for a maximum 1024 sessions. */
+    std::string db_create_config = CONNECTION_CREATE + ",session_max=1024";
```

Comment:
Why do we need to set the max number of sessions?

Reply:
By default its like 128 or something and if each thread has its own session we can easily go over that number. I bumped it to 1024 for now.